experimental/algorithm/LAGr_MaxFlow.c

```diff
@@ -92,7 +92,7 @@ static GrB_Info LG_global_relabel
     GrB_Vector src_and_sink,    // mask vector, with just [src sink]
     GrB_UnaryOp GetResidual,    // unary op to compute resid=capacity-flow
     // input/output:
-    GrB_Vector d,       // d(i) = height/label of node i
+    GrB_Vector *d,       // d(i) = height/label of node i
```

Comment:
Why does this need to be *d?  The contents of the vector can be changed if it is just a "GrB_Vector d" argument.  The *d would be needed only if there were a GrB_Vector_new (&d, ...) in the function.